ssl/quic/quic_stream_map.c

```diff
@@ -757,7 +757,8 @@ void ossl_quic_stream_map_remove_from_accept_queue(QUIC_STREAM_MAP *qsm,
     --qsm->num_accept;
 
     if ((max_streams_rxfc = qsm_get_max_streams_rxfc(qsm, s)) != NULL)
-        ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt);
+        if(!ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt);)
```

Comment:
The additional semicolon there is causing your CI failures

Reply:
Also, if you would be so kind as to sign a CLA so that we can accept this pull request, that would be much appreciated!